therapy/schemas.py

```diff
@@ -338,14 +339,38 @@ class ValueObjectDescriptor(BaseModel):
     extensions: Optional[List[Extension]]
 
 
+class ServiceMeta(BaseModel):
+    """Metadata regarding the therapy-normalization service."""
+
+    version: str
+    response_datetime: datetime
+
+    class Config:
+        """Enables orm_mode"""
+
+        @staticmethod
+        def schema_extra(schema: Dict[str, Any],
+                         model: Type['SourceMeta']) -> None:
+            """Configure OpenAPI schema"""
+            if 'title' in schema.keys():
+                schema.pop('title', None)
+            for prop in schema.get('properties', {}).values():
+                prop.pop('title', None)
+            schema['example'] = {
+                'version': '0.1.0',
+                'date': '2021-04-05T16:44:15.367831'
```

Comment:
```suggestion
                'response_datetime': '2021-04-05T16:44:15.367831'
```

Reply:
We should ask alex about `name` (if we choose to add) and I think `url` pointing to GitHub repo would be good.